improve shippingDate description (#282)
vtexdocs/dev-portal-content: docs/guides/Checkout/orderform-fields.md

|    dockId |    String  |   Dock ID.    |
|    courierName |    String  |   Courier name.   |
|    quantity |    Integer  |   Quantity.   |

> |    shippingEstimate |    String  | Total shipping estimate time. For instance, three business days is represented as `3bd`. The response for this field can include one of the following values: <ul><li><code>m</code> (minutes)</li><li><code>h</code> (hours)</li><li><code>bd</code> (business days)</li><li><code>d</code> (days)</li></ul> |

Reviewer: ```suggestion
|    shippingEstimate |    String  | Total shipping estimate time, represented by a number followed by a time unit. For instance, three business days is represented as `3bd`. The time unit can be one of the following: <ul><li><code>m</code> (minutes)</li><li><code>h</code> (hours)</li><li><code>bd</code> (business days)</li><li><code>d</code> (days)</li></ul> |
```

Reviewer: Outro ponto: sempre é um número inteiro? Ou pode ser `3.5bd` por exemplo?

Author: Não pode decimal, sempre inteiro.
